.gitignore

```diff
@@ -25,25 +25,8 @@ bld/
 [Ll]og/
 [Oo]utput/
 
-# localized repos that get cloned into scripts
```

Comment:
part of me says you might want to leave the old ones here for a little while?
otherwise, anyone whos run the scripts will suddenly have all these folders showing up as *additions * in git once they check out your new changes?

Reply:
sure